features/provider/dashboard/tabs.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Dashboard search bar
+  In order to navigate easily to products and backends
+  As a provider
+  I want to be able to filter them by name
+
+  Background:
+    Given a provider is logged in
+
+  @javascript
+  Scenario: Filtering products
+    When the products tab is selected
+    Then I should see "API" in the apis dashboard products tabs section
+    And I search for "AP" using the products search bar
+    Then I should see "API" in the apis dashboard products tabs section
+    And I search for "foo" using the products search bar
+    Then I should not see "API" in the apis dashboard products tabs section
+
+  @javascript
+  Scenario: Filtering backends
```

Comment:
not a big deal for CR2 but the 2 scenarios shouldn't have the same name

Reply:
What names would you give them?

Reply:
my bad :woman_facepalming: for checking too quickly :woman_facepalming: